pkgs/by-name/fo/follow/package.nix

```diff
@@ -0,0 +1,107 @@
+{
+  electron,
+  fetchFromGitHub,
+  imagemagick,
+  lib,
+  makeDesktopItem,
+  makeWrapper,
+  nodejs,
+  pnpm,
+  stdenv,
+}:
+let
```

Comment:
This is purely aesthetic, but we'd better just drop the let in block.

Reply:
Roger.